### DISCUSSION TITLE
spazio sul disco

### DISCUSSION THREAD

**Discussion by FlavioRadice** [indomus/forum#2905] [HUB: Home Assistant]
Ciao, sto utilizzando HA con Windows e non riesco a capire il motivo per cui lo spazio libero sta diminuendo rapidamente, c'è qualcosa che posso fare per pulire il database? Ho già rimosso tutti i backup lasciandone solo uno

aggiungo che ho anche provato questo comando, ma senza risultato
VBoxManage.exe modifyhd H:\HomeAssistatVM-Setup\haos_ova-9.3.vdi --compact
grazie

**Comment by snakuzzo** [comment]
https://indomus.it/formazione/configurare-correttamente-recorder-history-e-logbook-su-home-assistant/

**Reply by FlavioRadice** [reply]
grazie, ho provato ad aggiungere queste righe nel file configuration.yaml ma non sembra sia cambiato nulla
non sono un super esperto...

recorder: 
  include:
    domains:
      - sensor
      - switch
  purge_keep_days: 7
  auto_repack: true

logbook:

history:
